### PR TITLE
QAE-341 - E2E test case for primary header background gradient

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for primary header background gradient

### Screenshots

https://share.getcloudapp.com/NQuxedp6

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			'hb-header-bg-obj-responsive': {
+				desktop: {
+					'background-color': 'linear-gradient(135deg, rgb(6, 147, 227) 31%, rgb(155, 81, 224) 64%)',
+					'background-repeat': 'no-repeat',
+					'background-position': 'left top',
+					'background-size': 'contain',
+					'background-attachment': 'fixed',
+					'background-type': 'gradient',
+				},
+				tablet: {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js:16]: color should be different for responsive mode

File: tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+					'background-color': 'linear-gradient(135deg, rgb(6, 147, 227) 31%, rgb(155, 81, 224) 64%)',
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js:17]: BG color should be different for responsive mode

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+					'background-attachment': 'fixed',
+					'background-type': 'gradient',
+				},
+			},
+		};
+		await setCustomize( primaryHeader );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '#ast-desktop-header .ast-primary-header-bar' );
```
[the remark this answers is not in the document]
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js:38]: checking

[the remark this answers is not in the document]
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/header-types/primary-header/background-gradient.test.js]: Changed selector

**Review by GauravKhupse — APPROVED** [review]

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**